From b68c1d23f4ec93cc701b7df03fa7bc60a55aacad Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 22 Jan 2019 15:55:10 +0100
Subject: [PATCH 5/6] Revert "build: only write build_number.txt when
 BUILD_NUMBER has changed"

This reverts commit cdc6daea44f702766f2c55c2a7da4e28029a4c91.

Change-Id: I76d20f4eabce21103cf0e2220c97dc656924ca00
---
 core/main.mk | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index 4373e0481..8e6a71c89 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -63,11 +63,9 @@ include $(BUILD_SYSTEM)/clang/config.mk
 # Write the build number to a file so it can be read back in
 # without changing the command line every time.  Avoids rebuilds
 # when using ninja.
-BUILD_NUMBER_FILE := $(OUT_DIR)/build_number.txt
-ifneq ($(shell cat $(BUILD_NUMBER_FILE)),$(BUILD_NUMBER))
 $(shell mkdir -p $(OUT_DIR) && \
-    echo -n $(BUILD_NUMBER) > $(BUILD_NUMBER_FILE))
-endif
+    echo -n $(BUILD_NUMBER) > $(OUT_DIR)/build_number.txt)
+BUILD_NUMBER_FILE := $(OUT_DIR)/build_number.txt
 
 ifeq ($(HOST_OS),darwin)
 DATE_FROM_FILE := date -r $(BUILD_DATETIME_FROM_FILE)
-- 
2.17.1

